lib/susedistribution.pm

```diff
@@ -490,7 +490,7 @@ sub init_consoles {
                 hostname => get_required_var('SUT_IP'),
                 password => $testapi::password,
                 user     => 'root',
-                serial   => 'mkfifo /dev/sshserial; tail -fn +1 /dev/sshserial',
+                serial   => 'rm -f /dev/sshserial; mkfifo /dev/sshserial; while true; do cat /dev/sshserial; done',
```

Comment:
Great that you found this! I am a bit concerned that this can be inefficient compared to `tail -fn +1`. I wonder if we can tweak one of the two approaches to do what we want but in an efficient way…

Reply:
Normally the cat command stay in wait(blocked) state, i think the performance can be accepted.
For sure we can investigate more efficient way...